guides/common/modules/proc_formatting-hammer-output.adoc

```diff
@@ -4,28 +4,30 @@
 = Formatting Hammer output
 
 [role="_abstract"]
-You can modify the default formatting of the output of `hammer` commands to simplify the processing of this output by other command line tools and applications.
-For example, you can list organizations in a CSV format with a custom separator, for example, a semicolon:
+You can modify the default formatting of the output of Hammer commands to simplify the processing of this output by other command line tools and applications.
 
-[options="nowrap", subs="verbatim,quotes,attributes"]
-----
-$ hammer --csv --csv-separator ";" organization list
-----
-Output in CSV format is useful for example when you need to parse IDs and use them in a *for* loop.
-
-Several other formatting options are available with the `--output` option:
-
-[options="nowrap", subs="+quotes,attributes"]
-----
-$ hammer --output _output_format_ organization list
-----
-
-Replace _output_format_ with one of:
+Hammer provides several output formats:
 
 * `table` {endash} generates output in the form of a human readable table (default).
 * `base` {endash} generates output in the form of key-value pairs.
 * `yaml` {endash} generates output in the YAML format.
 * `csv` {endash} generates output in the Comma Separated Values format.
+Output in CSV format is useful for example when you need to parse IDs and use them in a *for* loop.
```

Comment:
I do that too and every time I do, I feel a bit ashamed of parsing CSV via CLI. IMO the better solution is to rely on JSON output and then using tools such as `jq` or even Python/Ruby/any scripting language that can parse JSON.